wf/ja-jp/Data/Tocs/wf__toc_wf_root_Chunk5.js

```diff
@@ -1 +1 @@
{i:[34],t:['数値とテキスト'],b:['']},'/Content/Topics/Workflows/learn/about-parent-flows.htm':{i:[24],t:['親フローとその他のフロータイプ'],b:['']},'/Content/Topics/Workflows/learn/about-save-data-with-flow-history.htm':{i:[38],t:['フローデータの保存'],b:['']},'/Content/Topics/Workflows/learn/about-streaming.htm':{i:[19],t:['アクションカードによるデータのストリーミング'],b:['']},'/Content/Topics/Workflows/learn/about-tables.htm':{i:[32],t:['テーブル'],b:['']},'/Content/Topics/Workflows/learn/about-updated-deprecated-cards.htm':{i:[21],t:['アップグレードされたカードと廃止予定のカード'],b:['']},'/Content/Topics/Workflows/learn/learn-workflows.htm':{i:[10],t:['Workflowsの概要'],b:['']},'/Content/Topics/Workflows/templates/add-template.htm':{i:[1474],t:['Workflows環境へのテンプレートの追加'],b:['']},'/Content/Topics/Workflows/templates/templates-list.htm':{i:[1473],t:['利用可能なWorkflowsテンプレート'],b:['']},'/Content/Topics/Workflows/templates/workflows-templates.htm':{i:[1472],t:['テンプレート'],b:['']},'/Content/Topics/Workflows/templates/workflows-usecases.htm':{i:[1475],t:['ソリューションパック'],b:['']},'/Content/Topics/Workflows/use-case-custom-user-notifications-crm.htm':{i:[1539],t:['カスタムエンドユーザー通知の送信'],b:['']},'/Content/Topics/Workflows/use-case-identity-proofing.htm':{i:[1540],t:['サードパーティサービスを使用した本人確認の実施'],b:['']},'/Content/Topics/Workflows/use-case-lifecycle-event-notification.htm':{i:[1537],t:['ライフサイクルアクティビティ発生時の通知の送信'],b:['']},'/Content/Topics/Workflows/use-case-lifecycle-event-reporting.htm':{i:[1536],t:['ライフサイクルアクティビティの報告'],b:['']},'/Content/Topics/Workflows/use-case-provision.htm':{i:[1534],t:['プロビジョニングとプロビジョニング解除'],b:['']},'/Content/Topics/Workflows/use-case-sync-customer-crm.htm':{i:[1538],t:['顧客IDとCRMおよびマーケティングツールとの同期'],b:['']},'/Content/Topics/Workflows/use-case-time-context-entitlement.htm':{i:[1535],t:['時間およびコンテキストベースのID権限の変更'],b:['']},'/Content/Topics/Workflows/use-case-time-inactivity-user-management.htm':{i:[1541],t:['時間に基づく条件によりユーザーのIDに対処する'],b:['']},'/Content/Topics/Workflows/use-cases-workflows-learn-about.htm':{i:[1533],t:['ユースケース'],b:['']},'/Content/Topics/Workflows/workflow-connect-your-applications.htm':{i:[40],t:['アプリケーションの接続'],b:['']},'/Content/Topics/Workflows/workflows-build-a-flow.htm':{i:[46],t:['フローの構築とテスト'],b:['']},'/Content/Topics/Workflows/workflows-capabilities.htm':{i:[11],t:['Workflowsの機能'],b:['']},'/Content/Topics/Workflows/workflows-data-deletion.htm':{i:[1548],t:['Workflowsでのデータの削除'],b:['']},'/Content/Topics/Workflows/workflows-elements.htm':{i:[12],t:['Workflowsの要素'],b:['']},'/Content/Topics/Workflows/workflows-error-types.htm':{i:[1547],t:['Workflowsでよくあるエラー'],b:['']},'/Content/Topics/Workflows/workflows-known-issues.htm':{i:[1546],t:['Workflowsの既知の問題'],b:['']},'/Content/Topics/Workflows/workflows-limits-learn-about.htm':{i:[1542],t:['ベストプラクティスとシステム制限'],b:['']},'/Content/Topics/Workflows/workflows-main.htm':{i:[1],t:['Okta Workflows'],b:['']},'/Content/Topics/Workflows/workflows-resources.htm':{i:[1549],t:['リソース'],b:['']},'/Content/Topics/Workflows/workflows-system-limits.htm':{i:[1545],t:['Workflowsのシステム制限'],b:['']}});
```

Comment:
@pengyao-okta We have a lot of TOC entries for connector actions that are in the `EN (JA)` pattern. But this convention is not upheld in the TOC consistently, and these entries are not UI controls anyway. 
I'd recommend we fix these on the next translation run -- basically, make it policy to translate strings in the TOC JS files (anything that isn't a proper noun/term that should always be EN).